### PR TITLE
Fixing not working sass imports and go to webIndexPlugin

### PR DESCRIPTION
**Pull request by kfiku**
Hi,

This repo stops to work propellery.
When you run `npm i` and `npm start` app will start but without sass / css and with error:
```
Uncaught Error: Cannot find module 'fuse-box-css'
    at Module._resolveFilename (module.js:470)
    at Function.Module._resolveFilename (/media/dev/www/github/ele-fuse/node_modules/electron/dist/resources/electron.asar/common/reset-sear…:35)
    at Function.Module._load (module.js:418)
    at Module.require (module.js:498)
    at require (internal/module.js:20)
    at u (app.js:487)
    at f (app.js:487)
    at c (app.js:487)
    at w.require (app.js:487)
    at Number.<anonymous> (app.js:29)
```

I fix this problem, and test it on mac and ubuntu linux.

### CODE REVIEW

**Comment by nchanged** [comment]
You should have just upgraded fuse-box ;-) 
I don't think loadURL is a good option because that server is for development. Try upgrading fuse-box and see if it helps.

**Comment by kfiku** [comment]
Upgrating don't work, you need to add ` + fuse-box-css` to .instructions(" > [index.ts]"); to make it work.

I can change it to not work with WebIndex, so it will work also without hrm, but this repo is not very production ready ;) I think it is more like example how to use fuse and electron.

What you think?

**Comment by nchanged** [comment]
Yep, I think WebIndex should be there. It's just i don't like openURL thingy as it is for development. You think you can help out and make it production ready? With Quantum? 
http://fuse-box.org/page/quantum

I think both targets can be set to `server` in Vanilla and Quantum as the latest FuseBox handles it all nicely

**Comment by kfiku** [comment]
You mean this: `mainWindow.loadURL('http://localhost:4445')` in electron.js?
I think i can do it, I'm not super experience in fuse-box (more webpack) and electron (i have made 2 apps) but i have 10+ years experience in web techs.

I have plans to contribute more to open source (fuse-box project is now my favorite) so this can by my starting point.

**Comment by nchanged** [comment]
@kfiku awesome!!!

**Comment by kfiku** [comment]
So what are plans for doing it? Do we just add prod and dev versions? Or add build process (electron-builder)? Add some more extra defaults linke in https://github.com/chentsulin/electron-react-boilerplate?

**Comment by nchanged** [comment]
Yep, adding task("dist") which will enable production mode
Like here https://github.com/fuse-box/fuse-box-examples/tree/master/examples/react-code-splitting

**Comment by kfiku** [comment]
Ok i make it move production ready.
Quantum plugin is comment out (we was talking about in on gitter) becouse its not compatible with electron target.

Let me know what you think

File: package.json [hunk 1]
```diff
@@ -5,7 +5,9 @@
     "main": "electron.js",
     "scripts": {
         "test": "echo \"Error: no test specified\" && exit 1",
-        "start": "node fuse"
+        "start": "export NODE_ENV=development && node fuse",
+        "dist": "export NODE_ENV=production && node fuse dist",
+        "prod": "export NODE_ENV=production && npm run dist && electron electron.js"
```
Comment (troynt) [inline package.json]:
```bash
NODE_ENV=development && node fuse
```
is enough, no export needed

**Comment by kfiku** [comment]
Please recheck. I also add move info to readme

**Comment by kfiku** [comment]
Hi guys, whats up with my pull request

**Comment by nchanged** [comment]
@kfiku sorry man, i will merge it as soon as i fix the css for electron (hopefully today)

**Comment by kfiku** [comment]
ok. If You wont some help with it, let me now. Im open to do some work on it

**Comment by nchanged** [comment]
@kfiku i will poke you later to test the new beta with this PR and Quantum, thanks!

**Comment by nchanged** [comment]
@kfiku let's test `fuse-box@2.2.2-beta.12` with Quantum target "electron", if CSS will work as expected, please uncomment the Quantum section, and i will merge

**Comment by kfiku** [comment]
@nchanged now its working

**Comment by nchanged** [comment]
Awesome! thanks! 👍

**Pull request merged**